template/deploy/helm/[[operator]]/templates/deployment.yaml.j2

```diff
@@ -17,8 +17,11 @@ spec:
     metadata:
       {{- with .Values.podAnnotations }}
       annotations:
+        internal.stackable.tech/image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
Looks like the value files are defined in the operator repos, and I noticed `image.tag` is not defined in opa-operator. It works there, so I don't see an issue here.

Reply:
Checking other operators, they don't specify image.tag either, so looks good :+1:

Reply:
Aye, that's what the `default` clause is for ^^